_data/past.yml

```diff
@@ -1142,3 +1142,15 @@
   dates: "May 25-26, 2018"
   url: http://balkanruby.com/
   twitter: balkanruby
+
+- name: RubyC
+  location: Kyiv, Ukraine
+  dates: "June 2-3, 2018"
+  url: http://rubyc.eu/
+  twitter: rubyc_eu
+
+- name: RubyKaigi 2018
```

Comment:
I think RubyKaigi 2018 needs to be before RubyC in this list since this is sorted in chronological order.